samples/packages/awscdk-typescript/test/component-distribution-staticwebsite.test.ts

```diff
@@ -0,0 +1,76 @@
+import "source-map-support/register";
+import * as cdk from "aws-cdk-lib";
+import * as cdf from "@openfabr/cdf";
+import {
+  PackageComponentConfig,
+  PackageCustomModule,
+  PackageGeneralConfig,
+  PackageInfraConfig,
+  PackageInfraPlanConstructs,
+  PackageNetworkConfig,
+  PackagePlanner,
+  PackageRelationConfig,
+  PackageServiceConfig,
+} from "../src/package-config";
+import configA from "./component-distribution-staticwebsite.config.json";
+import { ok, Result } from "neverthrow";
+import { Match, Template } from "aws-cdk-lib/assertions";
+
+class CustomModuleStub implements PackageCustomModule {
+  enhanceWith(
+    config: cdf.InfraConfig<
+      PackageGeneralConfig,
+      PackageNetworkConfig,
+      PackageComponentConfig,
+      PackageServiceConfig,
+      PackageRelationConfig
+    >,
+    result: cdf.InfraPlan<PackageInfraPlanConstructs>,
+    scope: any
+  ): Result<cdf.InfraPlanOutputs, cdf.PlanError> {
+    return ok(new Map<string, any>());
+  }
+}
+
+describe("A project", () => {
+  it("deploys a static website to S3 website bucket", () => {
+    const app = new cdk.App();
+
+    const orchestrator = new cdf.Orchestrator(
+      configA as PackageInfraConfig,
+      new PackagePlanner(),
+      []
+    );
+
+    const stack = new cdf.awscdk.ProjectStack(app, "ProjectStack", {
+      orchestrator,
+    });
```

Comment:
There's now a helper function call `initProjectStack` in the framework, which covers L39-47 combined